src/class-fields/initializer-eval/cls-decl-fields-indirect-eval.template

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2017 Valerie Young. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-performeval-rules-in-initializer
+path: language/statements/class/fields-indirect-
+name: indirect eval
+---*/
+
+var executed = false;
+class C = {
+  x = (0, eval)('executed = true; /*{ initializer }*/;';
```

Comment:
Missing a closing `)`


...Ok, I'm not going to be annoying and mark all of these—please fix each file :)